core/queryalgebra/evaluation/src/main/java/org/eclipse/rdf4j/query/algebra/evaluation/limited/iterator/LimitedSizeZeroLengthPathIteration.java

```diff
@@ -35,7 +35,7 @@
 	 * @param contextVar
 	 * @param bindings
 	 */
-	public LimitedSizeZeroLengthPathIteration(SimpleEvaluationStrategy evaluationStrategyImpl, Var subjectVar,
```

Comment:
What obstacles are there to having this take EvaluationStrategy instead of a specific implementation?